docs/docs/testing/typed-telemetry.mdx

```diff
@@ -143,7 +145,7 @@ startTelemetry(telemetryConfig);
 
 You can now run the app and see the telemetry being printed on your console.
 
-### Interpreting telemetry data {#telemetry_visualization}
+<Heading as="h3" id="telemetry_visualization">Interpreting telemetry data</Heading>
```

Comment:
Why the Heading tag? Is that preferred in .mdx?

Reply:
It's specific to Docusaurus. Their heading component will ensure that configured theme styles are applied to the heading, where otherwise we would have to specify `className="..."` in all of our `<h>` tags.

Docusaurus automatically applies these stylings to standard markdown headings, but not jsx ones.

Technically, Docusaurus supports the special ID syntax we were using here previously, but the plugin tooling we're using seems unable to parse that syntax. But now that I'm explaining this, I'm questioning my choice to migrate away from it here. I should investigate this more. There is probably a way to inject the necessary plugins to make this work. I think I'll split some of the aspects of this PR.

Reply:
Ha! Found a better solution. I've documented the better pattern in the docs README.